broker/src/main/java/org/apache/rocketmq/broker/transaction/queue/TransactionalMessageServiceImpl.java

```diff
@@ -100,13 +100,6 @@ private boolean putBackHalfMsgQueue(MessageExt msgExt, long offset) {
             msgExt.setCommitLogOffset(
                 putMessageResult.getAppendMessageResult().getWroteOffset());
             msgExt.setMsgId(putMessageResult.getAppendMessageResult().getMsgId());
-            log.info(
```

Comment:
Maybe make the log level to debug level is a better choice, because if we deleted all log, we will lose all hint to locate the issue.